### ISSUE TITLE
nativeCrashReportEnabled option is not working

### ISSUE THREAD

**Issue by VasiliyShevelDevPro** [DataDog/dd-sdk-reactnative#115]
**Describe what happened**
Crashes are not appear in the `UX Monitoring -> RUM Applications -> Total Crashes by Version` section when `nativeCrashReportEnabled` option is enabled.

**Steps to reproduce the issue:**
Set up Datadog using config
```
const config = new DdSdkReactNativeConfiguration(
  'REMOVED',
  __DEV__ ? 'dev' : 'prod',
  'REMOVED',
  true, // track User interactions (e.g.: Tap on buttons)
  true, // track XHR Resources
  true, // track Errors
);
config.nativeCrashReportEnabled = true;
DdSdkReactNative.initialize(config);
```

Crash app using IOS native code.
Example
```
NSMutableArray *tmp = [[NSMutableArray alloc] init];
NSObject *object = nil;
[tmp addObject:object];
```

**Describe what you expected:**
Crashes should appear in the `Total Crashes by Version` section.

**Additional context**
Version of the SDK - "dd-sdk-reactnative": "^1.0.0-beta3"

**Comment by xgouchet** [comment]
Hi @VasiliyShevelDevPro , 
As of today, crash reporting for iOS is only available as a public beta feature, and for native iOS applications only. Once this feature is fully complete and stable, we will make it available for React native as well.

**Comment by milkman4** [comment]
@xgouchet is there a timeline for it being fully complete and stable? We got an email about Beta being available for iOS but use this SDK with react native

**Comment by xgouchet** [comment]
I'm sorry but we don't have any ETA yet. We first want to make sure that this feature is totally stable and reliable for native apps, plus we need to add more work to make the Error Tracking in Datadog fully compatible with cross platform frameworks such as React Native

**Comment by vivekverma1993** [comment]
@xgouchet is crash reporting for Android is also a beta feature? we are testing datadog RUM in our react native app.

**Comment by xgouchet** [comment]
It's globally available on Android, and sold work as well for native crashes in React native as well.

**Comment by SparkleFaerieCoder** [comment]
Hi, I was wondering if there are any updates on this?

**Comment by xgouchet** [comment]
Hi @SparkleFaerieCoder , thanks for your interest, this is still a feature under development, we'll post an announcement as soon as it's ready.

**Comment by itajenglish** [comment]
You guys should update the documentation on this. To let users know it's not out for React Native just yet.

**Comment by itajenglish** [comment]
@xgouchet Also I'm a bit confused. Is it Crash Reporting and Error Tracking that isn't supported? The article you guys posted show error tracking working for React Native. 

https://www.datadoghq.com/blog/react-native-monitoring/#investigate-errors-faster

**Comment by xgouchet** [comment]
The SDK does report Errors in RUM, meaning that when you look at a session or explore RUM events, you'll see Errors tracked from your app. We don't provide yet crash reporting on iOS, and React Native errors are not yet integrated in our Error Tracking product (aggregating similar errors to see their trends across versions, OS versions etc…)

**Comment by ivoreis** [comment]
Hi team! Nice work with this react-native DD integration! 

I'm currently looking at the latest RC (RC7) and noticed that the iOS bridge being used is still `0.5.2` which doesn't include `nativeCrashReport`. Any chance to bump the dep to use https://github.com/DataDog/dd-bridge-ios/releases/tag/0.5.3 instead or is there any other work pending? 

Thanks in advance!

**Comment by xgouchet** [comment]
Hi @ivoreis , indeed that's a mistake, we're working on a new RC to include the iOS crash reporting

**Comment by ivoreis** [comment]
Thanks, @xgouchet! Looking forward to the next RC 👍🏼

**Comment by itajenglish** [comment]
🙌🏽

**Comment by louiszawadzki** [comment]
Hi everyone,
We've released version 1.0.0-rc8 yesterday which includes native iOS crash reporting.

Please note that If you are using distributed tracing, you now need to specify your hosts as first-party hosts in your SDK config after upgrading to this version:

```
config.firstPartyHosts = ['example.com'];
```

Feel free to open a new issue if you have any trouble :)
